playground/frontend/lib/modules/output/components/output_header/tab_header.dart

```diff
@@ -20,19 +20,17 @@ import 'package:flutter/material.dart';
 import 'package:playground/constants/sizes.dart';
 import 'package:playground/modules/output/components/output_header/output_placements.dart';
 
-import 'output_tabs.dart';
-
-class OutputHeader extends StatelessWidget {
+class TabHeader extends StatelessWidget {
   final TabController tabController;
   final bool showOutputPlacements;
```

Comment:
Since this widget is now contains general purpose headers, it can no longer contain this flag.
If the widget that creates this one needs the buttons, it should place this widget and the buttons to a `Row`.